examples/routing_service/routing_service_msql_adapter/cpp/rs_dll/CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+#
+# (c) 2018 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+#
+#  RTI grants Licensee a license to use, modify, compile, and create derivative
+#  works of the Software.  Licensee has the right to distribute object form
+#  only for use with RTI products.  The Software is provided "as is", with no
+#  warranty of any type, including any warranty for fitness for any purpose.
+#  RTI is under no obligation to maintain or support the Software.  RTI shall
+#  not be liable for any incidental or consequential damages arising out of the
+#  use or inability to use the software.
+
+
+cmake_minimum_required(VERSION 3.12)
+project (MsqlAdapter)
+
+if(DEFINED ENV{NDDSHOME})
+	# NDDSHOME is set
+else()
+    set(msg
+        "NDDSHOME environment variable must be set")
+    message(FATAL_ERROR ${msg})
+endif()
+
+if(NOT BUILD_SHARED_LIBS)
+    set(msg
+        "Static linking is not supported for this example. You can build it "
+        "using the -DBUILD_SHARED_LIBS=1 CMake parameter")
+    message(FATAL_ERROR ${msg})
+endif()
+
+if(NOT CONNEXTDDS_ARCH)
+    set(msg
+        "CONNEXTDDS_ARCH must be defined. For example, -DCONNEXTDDS_ARCH=x64Win64VS2017")
+    message(FATAL_ERROR ${msg})
+endif()
+
```

Comment:
This is not actually needed. The `FindConnextDDS.cmake` CMake script will take care of this kind of logic.

Reply:
Made the change. Will commit once all the changes are made.

Reply:
Done. Will commit all changes at once.